docs/DafnyRef/UserGuide.md

```diff
@@ -1225,15 +1225,16 @@ proof attempt multiple times after automatically making minor changes to
 the structure of the input or to the random choices made by the solver.
 If the resources used during these attempts (or the ability to find a
 proof at all) vary widely, we say that the verification of the relevant
-properties is _unstable_.
+properties is _highly variable_.
 
-#### 13.6.5.1. Measuring stability
+#### 13.6.5.1. Measuring proof variability
 
-To measure the stability of your proofs, start by using the
+To measure the variability of your proofs, start by using the
 `-randomSeedIterations:N` flag to instruct Dafny to attempt each proof
 goal `N` times, using a different random seed each time. The random seed
 used for each attempt is derived from the global random seed `S`
-specified with `-randomSeed:S`, which defaults to `0`.
+specified with `-randomSeed:S`, which defaults to `0` (which means use
+an arbitrary -- e.g. clock-based -- seed).
```

Comment:
I don't think this is true. At least on Dafny's side of the randomization process, `0` is the actual seed used, which helps ensure deterministic results. Dafny then sends this seed to Z3, as well, but I don't think it interprets `0` specially (though I could be wrong about that).